src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((el) => el)
+    .reduce((accumulator, value) => {
+      const result = value.split(':');
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider using a more descriptive name, such as `styleParts` or `declarationParts`, to indicate that this variable represents the components of a CSS declaration.